roofit/roofitcore/inc/RooFormulaVar.h

```diff
@@ -37,7 +37,9 @@ class RooFormulaVar : public RooAbsReal {
   virtual TObject* clone(const char* newname) const { return new RooFormulaVar(*this,newname); }
 
   inline Bool_t ok() const { return getFormula().ok() ; }
-
+  const TString& expression() const { return _formExpr; }
```

Comment:
```suggestion
  const char* expression() const { return _formExpr.Data(); }
```
`TString` is a legacy class, avoid using and returning it. I'd say `const char*` is the appropriate zero-copy option here.